content/03-reference/01-tools-and-interfaces/05-prisma-cli/02-command-reference.mdx

```diff
@@ -45,24 +53,107 @@ Examples
   $ prisma generate
 ```
 
+
+</CmdResult>
+
+</CodeWithResult>
+
+
 You can get additional help on any of the `prisma` commands by adding the `--help` flag after the command.
 
 ## Commands
 
-### init
+### <inlinecode>version</inlinecode>
```

Comment:
Would be great if this somehow could also mention `-v` as the short option of the command.
(No idea how though)